_tools/install-lookout-latest.sh

```diff
@@ -1,37 +1,26 @@
 #!/bin/bash
 #
 # Installs latest lookout SDK binary
-# Depends on GNU grep
-set -x
-
-#TODO(bzz): check for local cached version first
 
 OS="$(uname | tr '[:upper:]' '[:lower:]')"
+LATEST_TAG="$(git ls-remote -t --refs https://github.com/src-d/lookout \
```

Comment:
does it make sense to depend on installed `git` to use `lookout-sdk`?

Reply:
Adding to what @smacker just said, using this script is not a _requirement_ to use `lookout-sdk`, you can always go to the releases page and download the bin yourself. This is just a way to automate the download.

Reply:
fair enough.